app/helpers/samples_helper.rb

```diff
@@ -545,7 +545,7 @@ def samples_by_metadata_field(sample_ids, field_name)
   private
 
   def filter_by_time(samples, start_date, end_date)
-    samples.where("samples.created_at >= ? AND samples.created_at <= ?", start_date, end_date)
+    samples.where("samples.created_at >= ? AND samples.created_at <= ?", start_date.beginning_of_day, end_date.end_of_day)
```

Comment:
by the way, how we do treat time in idseq? do we normalize it to PST or UTC? do we try to localize it for the user?

Reply:
Not sure, but I believe it gets localized on view.